### PR TITLE
Removing compass

### PR DESCRIPTION
**Pull request by dpsxp**
Don't think there's necessary to add sass to this application, but since you're doing it please use node-sass. There's no need to add a ruby gem just to use some mixins.

### CODE REVIEW

**Comment by leonardorb** [comment]
Reviewing.

File: package.json [hunk 1]
```diff
@@ -3,8 +3,11 @@
   "version": "0.0.1",
   "description": "What's your GitHub score?",
   "author": "Leo <rb.leonardo@gmail.com>",
+  "engines": {
+    "node": "0.12.x"
+  }
```
Comment (leonardorb) [inline package.json]: Missing comma here.

File: package.json [hunk 2]
```diff
@@ -13,15 +16,12 @@
...
     "gulp": "^3.8.11",
-    "gulp-util": "^3.0.4",
     "gulp-coffee": "^2.3.1",
-    "gulp-compass": "^2.0.4",
-    "chai": "^2.3.0",
+    "gulp-sass": "^2.0.1",
+    "gulp-util": "^3.0.4",
     "mocha": "^2.2.5"
-  },
-  "scripts": {
```
Comment (leonardorb) [inline package.json:24]: Thanks for removing this dupe.

**Comment by leonardorb** [comment]
All good, except that missing comma.
After the fix, will wait for the green build on Travis and I'll merge.
Thanks dude! :metal:

**Comment by leonardorb** [comment]
And also, we'll need to update our `.travis.yml`.
https://github.com/leonardorb/github_score/blob/master/.travis.yml#L6

**Comment by dpsxp** [comment]
All done. Looks like everything is green again. :beer:

**Comment by leonardorb** [comment]
:beers:

**Pull request merged**